crates/eframe/Cargo.toml

```diff
@@ -131,7 +131,7 @@ image = { version = "0.24", default-features = false, features = [
   "png",
 ] } # Needed for app icon
 raw-window-handle.workspace = true
-winit = { version = "0.28.1", default-features = false }
+winit = { version = "0.29.4", default-features = false, features = ["rwh_05"] }
```

Comment:
why are we using raw-window-handle 0.5 instead of 0.6?

Reply:
It might make sense to upgrade raw-window-handle as a separate PR, to limit scope (and do we need to await glutin updating to rwh 0.6 as well)?

Reply:
Glutin hasn't upgraded yet: https://github.com/rust-windowing/glutin/pull/1582